services/notifications/service/auth_service.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"errors"
+	"github.com/HackIllinois/api/common/apirequest"
+	"github.com/HackIllinois/api/services/checkin/config"
+	"github.com/HackIllinois/api/services/checkin/models"
```

Comment:
Don't have cross service imports. The notifications config should be used instead of the checkin config and the required models should be copied into the notification service's model folder.